integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml

```
@@ -7,7 +7,7 @@ networks:
       name: apim_and_cc
 services:
   router:
-    image: wso2/choreo-connect-router:1.1.0-alpha
+    image: wso2/choreo-connect-router:1.1.0-alpha2-SNAPSHOT
```

docker image name should not be snapshots
```suggestion
    image: wso2/choreo-connect-router:1.1.0-alpha2
```